package/python-kiwi-keg.spec

```diff
@@ -32,32 +31,39 @@ Summary:        KEG - Image Composition Tool
 License:        GPL-3.0-or-later
 Source:         keg-%{version}.tar.gz
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
-BuildRequires:  %{python_module Jinja2}
-BuildRequires:  %{python_module Sphinx}
-BuildRequires:  %{python_module base >= 3.6}
 BuildRequires:  %{python_module setuptools}
-BuildRequires:  fdupes
-BuildArch:      noarch
-Requires:       python-Jinja2
-Requires:       python-PyYAML
-Requires:       python-docopt
-Requires:       python-schema
-Requires:       python3-kiwi >= 9.21.21
-%if %python_version_nodots < 37
-Requires:       python-iso8601
+BuildRequires:  %{python_module Sphinx}
+BuildRequires:  %{python_module Jinja2}
+BuildRequires:  %{python_module schema}
+BuildRequires:  %{python_module pytest}
+BuildRequires:  %{python_module docopt}
+BuildRequires:  %{python_module PyYAML}
+%if 0%{?suse_version} > 1500
```

Comment:
Shouldn't this be `< 15400`. Python 3.11 is only available in SLE 15 SP4 and later. In cases where build in the build service is enabled for 15 SP4 or earlier we'd fall back to `python3-kiwi` For everything else, i.e. Factory and SLE 15 Sp4 and later we'd pick up the expansion from the `python_module` macro.

Reply:
good point, I added the suggested version constraint. Please also note that we have python3-kiwi for the default python in TW and changing that will require an upstream conversation. That will not be so easy because that .spec file is also used by other distributors. It might be needed that we add a suse specific patch in this case. Because of that I still kept the "> 1500" version constraint

Reply:
I made a kiwi fix to address this:

* https://github.com/OSInside/kiwi/pull/2581

Reply:
Once the kiwi in SLE is out and the kiwi in TW we don't need special sauce here.

Reply:
and here is the patch for kiwi upstream regarding the versioned provides from someone who really knows how it should be done :) https://github.com/OSInside/kiwi/pull/2583